app/main/views/drafts.py

```diff
@@ -170,7 +171,8 @@ def fetch_draft_service(draft_id):
 
     return jsonify(
         services=draft.serialize(),
-        auditEvents=last_audit_event.serialize(include_user=True)
+        auditEvents=last_audit_event.serialize(include_user=True),
+        isValid=is_service_data_valid(draft)
```

Comment:
Can we return the validation errors instead/in addition to this? Frontend can display two separate messages instead of the "Mark as complete" button: about missing answers and about specialists/outcomes, it would be nice if we could eventually derive both from the API response